src/App.vue

```diff
@@ -5,11 +5,22 @@
 </template>
 
 <script lang="ts">
+import { useColorMode } from '@vueuse/core'
+
+const mode = useColorMode({
+  emitAuto: true
```

Comment:
Looks like `emitAuto` is deprecated: https://github.com/vueuse/vueuse/pull/2980/files#diff-5d628cd173c95cd5c9a1a70e3299bf4d58cceb8bcc22a31baf0cf7793f9db740R82